README.md

```diff
@@ -42,6 +42,7 @@ Databases on SQL Server, Azure SQL Database, or Azure SQL Managed Instance which
 ## Known Issues
 - The table used by a SQL binding or SQL trigger cannot contain two columns that only differ by casing (Ex. 'Name' and 'name'). 
 - Non-CSharp functions using SQL bindings against tables with columns of data types `BINARY` or `VARBINARY` need to map those columns to a string type. Input bindings will return the binary value as a base64 encoded string. Output bindings require the value upserted to binary columns to be a base64 encoded string.
+- SQL Bindings against tables with columns of data types `GEOMETRY` and `GEOGRAPHY` are not supported.
```

Comment:
Would it be good to include the issue link here for users to go to?

Reply:
Would be good thing to do for all these that we have tracking issues for actually. @lucyzhang929 do you have a bit of spare time to do that?

Reply:
Agreed if we can put the issue, that'd be great.

Also, please lowercase "Bindings" -> "bindings" (see other examples above).

Reply:
Yep I can link the issues